build-configs/integreat-e2e/index.js

```diff
@@ -14,18 +18,36 @@ const IntegreatE2eBuildConfig: BuildConfigType = {
     pushNotifications: false,
     introSlides: false,
     sentry: false
-  },
-  android: {
-    ...IntegreatTestCmsBuildConfig.android,
-    applicationId: 'tuerantuer.app.integreat',
-    googleServices: null
-  },
-  ios: {
-    ...IntegreatTestCmsBuildConfig.ios,
-    bundleIdentifier: 'de.integreat-app',
-    provisioningProfileSpecifier: 'match Development de.integreat-app',
-    googleServices: null
   }
 }
 
-export default IntegreatE2eBuildConfig
+const CommonIntegreatE2eBuildConfig: CommonBuildConfigType = {
```

Comment:
I think these should not be capitalized as they are just variables.

```suggestion
const commonIntegreatE2eBuildConfig: CommonBuildConfigType = {
```